doc/cookbook/source/examples/regression/linear_ridge_regression.rst

```diff
@@ -0,0 +1,49 @@
+=======================
+Linear Ridge Regression
+=======================
+
+A linear ridge regression model can be defined as :math:`y_i = \bf{w}^\top\bf{x_i}` where :math:`y_i` is the predicted value, :math:`\bf{x_i}` is a feature vector and :math:`\bf{w}` is the weight vector. We aim to find the linear function that best explains the data, i.e. minimizes the error or loss function :math:`E(\bf{w})` by finding appropriate :math:`\bf{w}`. One can show the solution can be written as:
```

Comment:
E(w) is not defined and should not be used. Just say "minimizes the squared loss plus a :math:`L_2` regularisation term"

(also remove: by finding appropriate w)